src/timearray.jl

```diff
@@ -13,11 +13,15 @@ abstract type AbstractTimeSeries{T,N,D} end
 
     TimeArray(timestamp, values[, colnames, meta=nothing])
     TimeArray(ta::TimeArray; timestamp, values, colnames, meta)
+    data = (datetime=[DateTime(2018, 11, 21, 12, 0), DateTime(2018, 11, 21, 13, 0)], col1=[10.2, 11.2], col2=[20.2, 21.2], col3=[30.2, 31.2])
+    ta = TimeArray(data; timestamp=:datetime, meta="Example")
```

Comment:
I will just put the function prototype here.
`TimeArray(data::NamedTuple; timestamp, meta)`

and add a `Examples` section, like the docstring in Base
```
# Examples
date = ...
ta = ...
```